crates/aptos/src/common/init.rs

```diff
@@ -114,13 +114,15 @@ impl CliCommand<()> for InitTool {
             network
         } else {
             eprintln!(
-                "Choose network from [devnet, testnet, local, custom | defaults to devnet]"
+                "Choose network from [devnet, testnet, local, custom | defaults to devnet]. For testnet, start over and run movement init --skip-faucet"
             );
             let input = read_line("network")?;
             let input = input.trim();
             if input.is_empty() {
                 eprintln!("No network given, using devnet...");
                 Network::Devnet
+            } else if input.eq_ignore_ascii_case("testnet") && self.skip_faucet {
+                Network::Testnet
             } else {
                 Network::from_str(input)?
             }
```

Comment:
I think you can remove this special case and instead forbid "testnet" without `--skip-faucet` by adding another check after line 127:

```rust
if network == Network::Testnet && !self.skip_faucet {
    return Err(todo!("report as appropriate"));
}
```